crates/primitives/gateway/src/test.rs

```diff
@@ -0,0 +1,24 @@
+
+
+
+
+
+
+#[cfg(test)]
+mod tests {
```

Comment:
remove that inner module since we're already in a test module
use #![cfg(test)] at the top of the file instead